controllers/movimiento_detalle.go

```diff
@@ -327,3 +331,59 @@ func (c *MovimientoDetalleController) PublicarMovimientosDetalle() {
 	}
 	c.ServeJSON()
 }
+
+// InsertarMovimientosDetalle ...
+// @Title InsertarMovimientosDetalle
+// @Description post InsertarMovimientosDetalle se encarga de devolver crear los movimientos detalle correspondientes a las cuentas recibidas
+// @Param     body      body   []models.MovimientoDetalle  true   "Cuentas presupuestales con su respectivo movimiento proceso externo y el valor/saldo afectado"
+// @Success   201   {object}   map[string]interface{}
+// @Failure   403   body is empty
+// @router /insertarMovimientosDetalle [post]
+func (c *MovimientoDetalleController) InsertarMovimientosDetalle() {
+	defer errorctrl.ErrorControlController(c.Controller, "CrearMovimientosDetalle2")
+
+	var movimientosDetalle []models.MovimientoDetalleInsertar
+	var movimientosRespuesta []models.MovimientoDetalle
+
+	if err := json.Unmarshal(c.Ctx.Input.RequestBody, &movimientosDetalle); err != nil {
+		panic(errorctrl.Error("CrearMovimientosDetalle2 - json.Unmarshal(c.Ctx.Input.RequestBody, &movimientosDetalle)", err, "404"))
```

Comment:
Acá iría el nombre de la función
```suggestion
		panic(errorctrl.Error("InsertarMovimientosDetalle - json.Unmarshal(c.Ctx.Input.RequestBody, &movimientosDetalle)", err, "404"))
```